git_machete/github.py

```diff
@@ -148,7 +151,7 @@ def __check_pr_already_created(pull: GitHubPullRequest, pull_requests: List[GitH
 
 
 def create_pull_request(org: str, repo: str, head: str, base: str, title: str, description: str, draft: bool) -> GitHubPullRequest:
-    token: Optional[str] = _get_token_from_env()
```

Comment:
WHOOPS

Reply:
A regression that happened on https://github.com/VirtusLab/git-machete/pull/181/files#diff-ede0d58ccd3510984ee2caca86265a8d8df90c0c108e1a8208611a524cc1b66a ... I didn't notice that

Reply:
Which is a lesson that `github` command should be tested - also against failure scenarios - before it's released... I've moved https://github.com/VirtusLab/git-machete/issues/178 to the current milestone... @rifel123 for you to research the options on how this can be tested